### ISSUE TITLE
Panel should announce title when opened

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode#77063]
**Environment Details:**
VSCode Version : 1.36.1 (user setup)
OS Version :Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
Screen Reader Versions : NVDA2019.1.1

**Repro Steps:**
1) Open Visual studio code editor  
2) Press ctrl+shift+y
3) Validate that debugging messages show-up in Debug console
4) Start NVDA

**Actual:**
NVDA does not announce the pop up title as 'Debug console' when it is opened using ctrl+shift+y.

This issue is observed in Win10 + NVDA

**Expected:**
Screen reader should announce the pop up title as 'debug console' when it is opened using ctrl+shift+y.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not get that on which page it is being redirected.

**MAS Reference:
MAS 1.3.1 -** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_DebugConsole_ScreenReader_NVDA.pptx](https://github.com/microsoft/vscode/files/3376373/A11y_VSCode_DebugConsole_ScreenReader_NVDA.pptx)

**Comment by georgebatalinski** [comment]
@isidorn I will have a look at this

**Comment by isidorn** [comment]
@georgebatalinski great, thanks! Ping me on the PR once you create it

**Comment by isidorn** [comment]
To verify make sure that NVDA announces Debug Console when pressing `ctrl+shift+y`

**Comment by isidorn** [comment]
@mjbvz nice catch. I have only tested it in on the mac.
